FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -39,6 +39,7 @@
 #define SERVOFAULT 40
 
 #define MEASUREMENTSPEED 1.0 //The max speed at which we move the motors when taking measurements
+int ENCODER_READ_FREQUENCY_HZ = 100;
```

Comment:
Down the road I'd like to try increasing the clock speed on wire and see if we can get this up to 300 or so. It isn't really necessary, but I think it might improve accuracy slightly

Reply:
before I added this, we were running it at like 400-500Hz, which I think is excessive. My intuition is 50 will work just as good